.github/workflows/retrieval-validation.yaml

```diff
@@ -33,10 +33,11 @@ jobs:
           python-version: "3.10"
 
       - name: Configure aws credentials
-        uses: aws-actions/configure-aws-credentials@v3
+        uses: aws-actions/configure-aws-credentials@v4
         with:
           role-to-assume: arn:aws:iam::${{ secrets.AWS_ACCOUNT_ID }}:role/github_actions_dev
           role-session-name: github-actions
+          role-duration-seconds: 7200
```

Comment:
By default "AWS assume role" creates a temporary credential of <= 1hour. I increased the hard limit for the github actions role on AWS and also increased role duration here.